python/lsst/daf/butler/core/location.py

```diff
@@ -141,6 +141,20 @@ def path(self):
         """The path component of the URI."""
         return self._uri.path
 
+    @property
+    def ospath(self):
+        """Path component of the URI localized to current OS."""
+        return posix2os(self._uri.path)
+
+    @property
+    def relativeToPathRoot(self):
+        """Returns path relative to network location.
+
+        Effectively, this is the path property with posix separator stipped
+        from the left hand side of the path.
+        """
+        return self._uri.path.lstrip('/')
```

Comment:
This will not work for scheme-less URIs because they need to be converted to absolute paths before they can be converted to being relative to root.  Secondly you can do this without hard-coding "/" by using `PurePath` and `pathlib.PosixPurePath`. For non-schemeless you know that the path is absolute so you can do something like:

```python
p = PurePosixPath(self._uri.path)
return p.relative_to(p.root)
```

Reply:
Thanks!